bindings/py/tests/encoders/coordinate_encoder_test.py

```diff
@@ -0,0 +1,256 @@
+# ----------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2019, David McDougall
+#
+# The following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+# ----------------------------------------------------------------------
+
+"""Unit tests for Coordinate Encoder."""
+
+import pickle
+import numpy as np
+import unittest
+
+from nupic.bindings.sdr import SDR, Metrics
+from nupic.bindings.encoders import CoordinateEncoder, CoordinateEncoderParameters
```

Comment:
OT: can we have params as a public member, so we don't have to import both encoder and its params, instead: 
`import encoder`, and `args = Coord.parameters` ?